packages/react-static-plugin-css-modules/src/node.api.js

```diff
@@ -0,0 +1,33 @@
+import ExtractCssChunks from 'extract-css-chunks-webpack-plugin'
+
+export default (options = {}) => ({
+  webpack: (config, { stage }) => {
+    let loaders = []
+
+    const styleLoader = { loader: 'style-loader' }
+    const cssLoader = {
+      loader: 'css-loader',
+      options: Object.assign({
+        modules: true,
+        importLoaders: 1,
+      }, options),
+    }
+
+    if (stage === 'dev') {
+      // Dev
+      loaders = [styleLoader, cssLoader]
+    } else if (stage === 'node') {
+      cssLoader.loader = 'css-loader/locals'
```

Comment:
Should this also assign to the `loaders` array? Or maybe something like this in a single action:

```
      loaders = [
        { ...cssLoader, loader: 'css-loader/locals' }
      ]
```

Reply:
Hmm. Technically if it’s node it will always be an object. So we would only need to mutate the loader property, in this instance we are not returning and additional loaders. 

That being said, I likely missed the obvious that in fact we need to re state the loaders variables 🤦‍♂️ 

Yes I believe you are correct. This does indeed need to be updated

Reply:
I think after we get this cleared up, we can merge this.